packages/control/process.py

```diff
@@ -56,7 +56,7 @@ def process_algorithm_results(self) -> None:
                     else:
                         if cp.data.get.charge_state:
                             Pub().pub(
-                                f"openWB/set/chargepoint/{cp.num}/get/state_str", "Lädt... ")
+                                f"openWB/set/chargepoint/{cp.num}/get/state_str", "Ladung aktiv")
```

Comment:
```suggestion
                                f"openWB/set/chargepoint/{cp.num}/get/state_str", "Lädt...")
```
Nach Rücksprache mit dem Support soll dieser Text beibehalten werden. Die anderen Vorschläge sind gut.